MonoidalCategories/gap/MonoidalCategoriesTensorProductAndUnit.gd

```diff
@@ -0,0 +1,72 @@
+####################################
```

Comment:
I guess that these files (`MonoidalCategoriesTensorProductAndUnit*`) were created for purely technical reasons, and not for imposing a "hierachy", because otherwise,  I would expect these functions not being seperated from `TensorProductOnMorphisms` or `LeftUnitor/RightUnitor`. Is it possible to rectify this by either inserting these functions into the files `MonoidalCategories*`, or inserting the complementary methods for tensor products and unitors into this file?

If not, then at least, these files should have a more technical name that makes it easier to guess what it contains, like
`MonoidalCategoriesTensorProductOnObjectsAndUnitObject*`.

Reply:
The reason is indeed technical. As for the generation of `Toposes` I replaced `TensorProduct` -> `DirectProduct` I cannot have the content of the files `MonoidalCategoriesTensorProductAndUnit*` in `MonoidalCategories*`, since GAP will complain about multiple declarations. I am ok with renaming to `MonoidalCategoriesTensorProductOnObjectsAndUnitObject*`  if you agree with my argument.

Reply:
Yes, I understand and agree.